web-console/3-project-details/labels.md

```diff
@@ -0,0 +1,104 @@
+# Labels on Detail Pages
+
+## View Labels
+- Labels are currently displayed at the top of most detail pages (Deployments, Pods, Builds, Images, Storage, etc)
+- A new `Labels` section should be added to the bottom of the `Details` tab on all of these pages to allow for a more comprehensive labels area
+
+![Label 1](img/new-labels-00-view.png)
+
+## Add Labels
+![Label 1](img/new-labels-01-add.png)
+
+- A `Manage Labels` link should be displayed below the applied labels
+- Clicking the link will open a modal, displaying the labels that are already applied, as well as an option to add new labels.
+- Clicking `Add` will display a new row with two empty text fields for label Name and label Value.
+
+![Label 1](img/new-labels-02-add.png)
+
+- The save button should not be enabled until valid entries are made.
+
+![Label 1](img/new-labels-03-add.png)
+
+- Invalid characters will result in a field level error once the user changes focus away from the particular field.
+
+![Label 1](img/new-labels-03B-add.png)
+
+- Label names cannot have more than one value associated.
+- Attempting to enter a label name that has already been applied with a value will result in a field level error as well.
+
+![Label 1](img/new-labels-04-add.png)
+
+![Label 1](img/new-labels-05-add.png)
+
+- Once the user clicks `Save` the new label should appear at the bottom of the list of labels on the Details tab.
+
+## Edit Labels
+![Label 1](img/new-labels-01-edit.png)
+
+- The same `Manage Labels` link will be used for editing existing labels.
+- Users can change a name or value as desired, as long as the name is not already associated with another value.
+
+#### Implementation Details
+- Values can be edited in this view, but only the current object will receive the label change. In order to bulk change labels for multiple items (Deployments in this case), users must go to the summary page and multi-select items.
+- If there are any restricted labels that users are unable to change, the entire row should be disabled.
+
+## Remove Labels
+![Label 1](img/new-labels-01-delete.png)
+
+- The same `Manage Labels` link will be used for removing existing labels.
+- Clicking the x icon next to a label will remove that row and enable the save button.
+- Applying these changes will only remove the label from this specific object, not from the system entirely.
+
+![Label 1](img/new-labels-02-delete.png)
+
+# Labels on Summary Pages
+
+![Label 1](img/new-labels-01-manage.png)
+
+- Summary lists are provided for each major section such as Deployments, Pods, Builds, Images, Storage, etc.
```

Comment:
Similar to labels on Pod details pages, I wonder if labels for Pods are read-only. @spadgett?